website/__init__.py

```diff
@@ -16,7 +16,7 @@ def create_app():
     if ENV == 'dev':
         app.debug = True
         print("Using dev server")
-        app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:steven@localhost/novelt'
```

Comment:
I would suggest not to commit changes like this in the future, because it is for your own dev environment.